media/createSkill/createSkill.html

```diff
@@ -42,13 +42,21 @@ <h1>Create new skill</h1>
                             <option value='es-US'>Spanish (US)</option>
                         </select>
                         <br/><br/>
-                        <label for="skillName">Hosting option</label><br/>
+                        <label for="skillRuntime">Hosting runtime</label><br/>
                         <span>With Alexa-Hosted skills you get free hosting with AWS Lambda (up to the AWS Free Tier). <a href='https://developer.amazon.com/en-US/docs/alexa/hosted-skills/build-a-skill-end-to-end-using-an-alexa-hosted-skill.html'>Learn more</a></span>
                         <select id="runtime" name="runtime">
                             <option value='NODE_10_X'>Alexa-Hosted (Node.js)</option>
                             <option value='PYTHON_3_7'>Alexa-Hosted (Python)</option>
                         </select>
                         <br/><br/>
+                        <label for="skillRegion">Hosting region</label><br/>
+                        <span>For reduced latency, you can select a region where your skill code default endpoint is hosted. <a href='https://developer.amazon.com/en-US/docs/alexa/hosted-skills/build-a-skill-end-to-end-using-an-alexa-hosted-skill.html#regions'>Learn more</a></span>
+                        <select id="region" name="region">
```

Comment:
In case no region has been specified, will us-east-1 be chosen as the default?

Reply:
That's right, since it's the first option. Web console does it a bit different though. When you change the locale, the default region value is automatically changed to the best compatible value using this matrix : https://developer.amazon.com/en-US/docs/alexa/hosted-skills/build-a-skill-end-to-end-using-an-alexa-hosted-skill.html#regions. However, skill devs can change it manually. 

Do you think that would be useful to add here? I am worried that the matrix might not reflect the service side changes always and we may get out of sync. However, we can probably provide a `warning` message saying for a specific `locale`, we suggest using a specific `region`.

Reply:
Added the `onChangeEvent` function on `locale`, to set the default aws region.